app/src/main/java/de/danoeh/antennapod/adapter/FeedItemlistAdapter.java

```diff
@@ -153,7 +141,9 @@ public View getView(final int position, View convertView, ViewGroup parent) {
                     holder.episodeProgress.setProgress(((ItemAccess) itemAccess).getItemDownloadProgressPercent(item));
                     holder.published.setVisibility(View.GONE);
                 } else {
-                    holder.episodeProgress.setVisibility(View.GONE);
+                    if(media.getPosition() == 0) {
```

Comment:
I think this is preventing the EpisodeProgress from going away when we start playback of another episode.

It results in this:
![screenshot 2015-05-11 19 43 04](https://cloud.githubusercontent.com/assets/5216560/7577428/ff5ad766-f815-11e4-8f2f-df74d3808ab4.png)